pyproject.toml

```diff
@@ -168,7 +168,7 @@ exclude = ["docs*", "tests*"]
 [tool.setuptools.package-data]
 "pycontrails" = ["py.typed"]
 "pycontrails.models.emissions" = ["static/*.csv"]
-"pycontrails.models" = ["quantiles/*.npy"]
+"pycontrails.models" = ["quantiles/era5-quantiles.pq"]
```

Comment:
Remember to update paths here!

Reply:
1c61242657a9ccab0245473d6feaa4816e5fd7a2